### PR TITLE
Feature/log in users part2

### PR DESCRIPTION
**Pull request by LostFan**
Add login for current users

### CODE REVIEW

File: src/main/java/com/epam/idea/core/service/UserService.java [hunk 1]
```diff
@@ -22,4 +22,6 @@
 	 * @throws com.epam.idea.core.service.exception.UserNotFoundException If no user was found with the given id.
```
Comment (Hanna-Aliakseichykava) [inline src/main/java/com/epam/idea/core/service/UserService.java]: update comment, please: "with the given id" -> "with given email and...."

File: src/main/webapp/js/authentification/authenticationService.js [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+                });
+
+
+
+        }
+
+        function SetCredentials(email, password) {
+            var authdata = md5.createHash((email + ':' + password) || '');
+
+            $rootScope.globals = {
```
Comment (Hanna-Aliakseichykava) [inline src/main/webapp/js/authentification/authenticationService.js:36]: May be, it makes sense store on user side only id and required for displaying user data (email) (it is seems like password in not required later)?

File: src/main/java/com/epam/idea/core/service/impl/UserServiceImpl.java [hunk 3]
```diff
@@ -56,4 +56,9 @@ public User update(final long userId, final User source) {
 		target.updateWith(source);
 		return target;
 	}
+
```
Comment (Hanna-Aliakseichykava) [inline src/main/java/com/epam/idea/core/service/impl/UserServiceImpl.java:59]: I think that passwords in db should be saved in encrypted state.

Comment (Hanna-Aliakseichykava) [inline src/main/java/com/epam/idea/core/service/impl/UserServiceImpl.java:59]: And before comparison we should encrypt the inputted password as well

**Comment by Hanna-Aliakseichykava** [comment]
:+1:

**Comment by reshetnev** [comment]
:+1:

**Pull request merged**